src/MerchantCenter/PriceBenchmarks.php

```diff
@@ -101,15 +310,145 @@ public function get_summary(): array {
 			$job->schedule();
```

Comment:
Now that we're relying on the local DB being up to date for both the summary data and for the price benchmarks list, we should look at moving the responsibility for scheduling this job elsewhere and make this method only responsible for getting and formatting the summary data.

Reply:
I'm having a bit of trouble fully understanding how all of the pieces of the Jobs system is meant to work, but found that the `JobInitializer` class [has some special handling ](https://github.com/woocommerce/google-listings-and-ads/blob/c5c67ec46c7ad7218ecc554edae7e8e952375712/src/Jobs/JobInitializer.php#L43-L72)for certain types of jobs to register a hook for scheduling and another that automatically calls that hook once a day.

I've made that change in 6888aee and it seems to be working.

Reply:
Seems it set schedule job 3AM everyday.

https://github.com/user-attachments/assets/36ddc20d-7d50-41ed-9898-b7e5312b9bd7

Reply:
Correct. It seems like any jobs implementing the `RecurringJobInterface` will automatically schedule a 'start' event to run every day at 3am, but that event is only used to ensure the job is scheduled.

<img width="1509" alt="image" src="https://github.com/user-attachments/assets/915304b1-ee67-4523-85e0-f758941e22a3" />

You can see here that there are two scheduled jobs:

* `gla/jobs/update_merchant_price_benchmarks/start` calls the start hook which causes the `UpdateMerchantPriceBenchmarks::schedule()` method to be called.
* `gla/jobs/update_merchant_price_benchmarks/process_item` runs every 12 hours and calls the `UpdateMerchantPriceBenchmarks::process_items()` method.